lib/css/modules/_orangered.scss

```diff
@@ -1,21 +1,8 @@
-#NREMail {
-	width: 15px;
-	height: 10px;
-	margin-left: 4px;
-	background: center center no-repeat;
-
-	&.nohavemail {
-		background-image: url('../images/mailgray.png');
-	}
-
-	&.havemail {
-		background-image: url('../images/mail.png');
-	}
-}
-
-#NREMailCount {
-	line-height: 12px;
-	margin: 0 0 0 1px;
+.res-floater-visibleAfterScroll {
```

Comment:
<img width="155" alt="screen shot 2017-08-07 at 9 34 25 pm" src="https://user-images.githubusercontent.com/455632/29056492-37fc5dc4-7bb8-11e7-9acf-51029471691a.png">

Is ... less bad now.